python/llm/src/bigdl/llm/ggml/convert.py

```diff
@@ -0,0 +1,97 @@
+#
+# Copyright 2016 The BigDL Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
```

Comment:
Is this file copied or adapted from somewhere? Need to add original link (see https://github.com/intel-analytics/BigDL/blob/main/python/orca/example/learn/tf2/yolov3/predict.py#L18)

Reply:
> Is this file copied or adapted from somewhere? Need to add original link (see https://github.com/intel-analytics/BigDL/blob/main/python/orca/example/learn/tf2/yolov3/predict.py#L18)

Original links have been added.